alibi/explainers/tests/utils.py

```diff
@@ -96,6 +96,25 @@ def fashion_mnist_dataset():
     }
 
 
+def mnist_dataset():
```

Comment:
should these functions move to the `alibi_models` repo? When we load the data, we want to ensure that it is compatible with the model trained.

Reply:
Possibly, although I want to avoid too much fragmentation of test utility code between the two repos. We need to think about this a bit more.